mixin.scss

```diff
@@ -0,0 +1,32 @@
+@function bk-calculateTypeOffset($line-height, $font-size, $descdender-height-scale){
+  $line-height-scale: $line-height / $font-size;
+  $line-height-scale: (($line-height-scale - 1) / 2) + $descdender-height-scale;
+  @return $line-height-scale;
+}
+
+@mixin bk-render($font-size, $line-height, $type-offset){
+  font-size: $font-size * 1px;
+  line-height: $line-height * 1px;
+  transform: translateY( $type-offset * 1em );
+}
```

Comment:
My SASS experience is non-existent so excuse my ignorance, but any reason you dont place these functions inside the `basekick` mixin? Are they global when they sit out here?